openssl/src/md_ctx.rs

```diff
@@ -118,6 +118,17 @@ impl MdCtx {
 }
 
 impl MdCtxRef {
+    /// Performs a copy of this instance of `MdCtxRef` to another `MdCtxRef`
+    #[corresponds(EVP_MD_CTX_copy_ex)]
+    #[inline]
+    pub fn copy(&self, other: &mut MdCtxRef) -> Result<(), ErrorStack> {
```

Comment:
Nit, I'd probably reverse the arguments to both match the OpenSSL function signature and line up with the standard library's `clone_from`.

LGTM otherwise.